src/classes/CRLP_DefaultConfigBuilder.cls

```diff
@@ -466,25 +466,25 @@ public class CRLP_DefaultConfigBuilder {
             'General_Accounting_Unit__c.Last_Allocation_Date__c' =>
                     new List<String>{ 'GAU: Last Allocation Date', 'Date of the last Opportunity with an Allocation to this GAU' },
             'General_Accounting_Unit__c.Largest_Allocation__c' =>
-                    new List<String>{ 'GAU: Largest Allocation', 'Largest amount allocated to this GAU' },
+                    new List<String>{ 'GAU: Largest Allocation', 'Largest gift amount to this GAU' },
```

Comment:
@lparrott I'm confused. These are not the field, inline help text, and field description changes that @juliesnow and I have been discussing adding into this PR. These are changes to customizable rollups, which should not be in this PR.

Reply:
I think we're all confused, these are the changes we thought you were talking about above, which I was told have been approved by all parties.